File: Java/AreaOfCircle.java
import java.util.*;
class AreaOfCircle {
    public static void main(String[] args) {
        float r, a;
        final float PI = 3.14F;
        Scanner sc = new Scanner(System.in);        
        System.out.print("Enter radius: ");
        r = sc.nextFloat();
        a = PI * r * r;
        System.out.println("The area is " + a + " sq. units.");
    }
}

File: CPP/AreaOfCircle.cpp
#include<iostream>              //1st diff
using namespace std;
#define PI 3.14
int main()
{
    float r, a;
    cout<<"Enter radius: ";     //2nd diff
    cin>>r;                     //3rd diff
    a = PI * r * r;
    cout<<"The area of circle is "<<a<<" sq. units"<<endl;
    return 0;
}